### PR TITLE
WIP Add rosa hypershift workflow to docs

### PR DESCRIPTION
**Pull request by ascerra**
Now that we are expanding into testing layered products on rosa hypershift we want to make the workflow that we will be using more known.

### CODE REVIEW

File: docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md [hunk 1]
```diff
@@ -238,9 +238,64 @@ We have the option to set an env variable in the config to use AWS spot instance
...
+
+Usage:
+
+ - There are many ways to use this workflow, so just to show a simple example we will show requirements to test ROSA Hypershift staging. (Please see the [Environment](https://steps.ci.openshift.org/workflow/rosa-aws-sts-hypershift#environment) section of the workflow to learn about these vars and more).
+
+Files created in the $SHARED_DIR
+
+ - The api.login file holds the username and password to login to the ROSA hypershift cluster.
+
+How to Access the ROSA Hypershift cluster:
```
Comment (calebevans) [inline docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md]: Can we move this down to this section or move it and link to this section here?

https://github.com/CSPI-QE/ocp-ci-docs/blob/main/docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md#accessing-the-test-cluster

**Review by calebevans — COMMENTED** [review]
Just a quick suggestion for organizations sake.

File: docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md [hunk 2]
```diff
@@ -238,9 +238,64 @@ We have the option to set an env variable in the config to use AWS spot instance
 > 
 > Spot instances are not to be used in production.
 
-##### Important Workflow.Next<!-- omit from toc -->
+##### firewatch-rosa-aws-sts-hypershift<!-- omit from toc -->
```
Comment (smpsgill) [inline docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md]:
```suggestion
##### `firewatch-rosa-aws-sts-hypershift`<!-- omit from toc -->
```
```

**Review by shakyav — CHANGES_REQUESTED** [review]
used to created Jira issues based on failures  ->  used to create Jira issues based on failures

File: docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md [hunk 3]
```diff
@@ -238,9 +238,64 @@ We have the option to set an env variable in the config to use AWS spot instance
...
+
+ - This workflow also has a post section that will cleanup all AWS resources that were created as part of the workflows pre section ([Post steps](https://steps.ci.openshift.org/workflow/firewatch-rosa-aws-sts-hypershift#post)).
+
+Usage:
+
+ - There are many ways to use this workflow, so just to show a simple example we will show requirements to test ROSA Hypershift staging. (Please see the [Environment](https://steps.ci.openshift.org/workflow/rosa-aws-sts-hypershift#environment) section of the workflow to learn about these vars and more).
+
+Files created in the $SHARED_DIR
+
+ - The api.login file holds the username and password to login to the ROSA hypershift cluster.
```
Comment (chetna14manku) [inline docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md:267]: I see there are many files created in `$SHARED_DIR`, I think we can attach a screenshot here and move this above line to `Access the ROSA Hypershift cluster` section, wdyt?

File: docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md [hunk 4]
```diff
@@ -238,9 +238,64 @@ We have the option to set an env variable in the config to use AWS spot instance
...
+
+AWS cleanup done by workflow:
+
+ - This workflow also has a post section that will cleanup all AWS resources that were created as part of the workflows pre section ([Post steps](https://steps.ci.openshift.org/workflow/firewatch-rosa-aws-sts-hypershift#post)).
+
+Usage:
+
+ - There are many ways to use this workflow, so just to show a simple example we will show requirements to test ROSA Hypershift staging. (Please see the [Environment](https://steps.ci.openshift.org/workflow/rosa-aws-sts-hypershift#environment) section of the workflow to learn about these vars and more).
+
+Files created in the $SHARED_DIR
```
Comment (chetna14manku) [inline docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md]:
```suggestion
Files created in the $SHARED_DIR after cluster provision
```

**Review by shakyav — COMMENTED** [review]
can be used to created Jira issues based on failures -> can be used to create Jira issues based on failures

File: docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md (hunk 3 above)
Comment (ascerra) [reply docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md:267]: We are adding a screen shot to show what's in the $SHARED_DIR on [line 269 here ](https://github.com/CSPI-QE/ocp-ci-docs/pull/24/files#diff-d4ac4b95660e5af0e5d2210d357ffd29b8bd01a3fb8e7497e45fc136845a766cR269)

**Review by smpsgill — APPROVED** [review]

File: docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md [hunk 5]
```diff
@@ -33,6 +33,8 @@
   - [Accessing the Test Cluster](#accessing-the-test-cluster)
     - [Access the API (`oc`)](#access-the-api-oc)
     - [Access the Console](#access-the-console)
+    - [Access the ROSA Hypershift cluster from cli:](#access-the-rosa-hypershift-cluster-from-cli)
+    - [Access the ROSA Hypershift cluster from the console:](#access-the-rosa-hypershift-cluster-from-the-console)
```
Comment (chetna14manku) [inline docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md]:
```suggestion
    - [Access the ROSA Hypershift cluster from cli](#access-the-rosa-hypershift-cluster-from-cli)
    - [Access the ROSA Hypershift cluster from the console](#access-the-rosa-hypershift-cluster-from-the-console)
```

**Review by chetna14manku — COMMENTED** [review]
Added a small change, otherwise all looks good to me

File: docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md (hunk 5 above)
Comment (ascerra) [reply docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md]: Done

**Review by calebevans — APPROVED** [review]
lgtm

**Pull request merged**